src/gsw_internal_funcs/mod.rs

```diff
@@ -1224,7 +1224,55 @@ gsw_linear_interp_SA_CT
 gsw_pchip_interp_SA_CT
 gsw_rr68_interp_SA_CT
 gsw_spline_interp_SA_CT
-gsw_gibbs_pt0_pt0
 gsw_gibbs_ice_part_t
 gsw_gibbs_ice_pt0
 */
+
+
+pub fn gibbs_pt0_pt0(sa: f64, pt0: f64) -> Result<f64> {
+
+    let sa: f64 = if sa < 0.0 {
+        if cfg!(feature = "compat") {
+            0.0
+        } else if cfg!(feature = "invalidasnan") {
+            return Ok(f64::NAN);
+        } else {
+            return Err(Error::NegativeSalinity);
+        }
+    } else {
+        sa
+    };
+
+    let x2 = GSW_SFAC * sa;
+    let x  = libm::sqrt(x2);
+    let y  = pt0 * 0.025;
+
+    let g03 = -24_715.571_866_078
```

Comment:
Just a not to myself to double check these coefficients againsta the scientific article/report.